Xamarin.PropertyEditing.Mac/Controls/Custom/FocusableBooleanButton.cs

```diff
@@ -3,7 +3,28 @@
 
 namespace Xamarin.PropertyEditing.Mac
 {
-	internal class FocusableBooleanButton : NSButton
+	internal class ProxyResponderButton : NSButton
+	{
+		public RowProxyResponder ProxyResponder { get; set; }
+
+		public override void KeyDown (NSEvent theEvent)
+		{
+			switch (theEvent.KeyCode) {
+			case (int)NSKey.Tab:
+				if (ProxyResponder != null) {
+					if (theEvent.ModifierFlags.HasFlag(NSEventModifierMask.ShiftKeyMask)) {
+						ProxyResponder.PreviousResponder ();
+					} else {
+						ProxyResponder.NextResponder ();
+					}
+				}
+				return;
```

Comment:
Shouldn't this happen only when there is a proxy responder?

```suggestion
					}
					return;
				}
```

Reply:
true like water

Reply:
fixed